lib/errors/index.ts

```diff
@@ -34,3 +35,12 @@ export interface NoRefImageError {
     currImg: ImageFile;
     refImg: ImageFile;
 }
+
+export interface InvalidRefImageError {
```

Comment:
Lets leave a comment here: "could be thrown if testplane >= X.Y.Z", like it is described higher, so we knew its only supported in testplane X.Y.Z+